bin/v-import-directadmin

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+# info: Import DirectAdmin backup to a new user
+#
+# example: v-import-directadmin /backup/backup.tar.gz
+#
+# Based on sk-da-importer
+# Credits: Maks Usmanov (skamasle), Jaap Marcus (jaapmarcus) and contributors:
+# Thanks to <https://github.com/Skamasle/sk_da_importer/graphs/contributors>
+
+# This script is provided whitout any warranty
+# Run at your own risk
+# Version 0.1
+# This script restore backups from DA to Hestiacp
+
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+if [ ! -e /usr/bin/rsync ] || [ ! -e /usr/bin/file ]; then
+	echo "#######################################"
+	echo "rsync not installed, try install it"
+	echo "This script need: rsync, file"
+	echo "#######################################"
+	if [ -e /etc/redhat-release ]; then
+		echo "Run: yum install rync file"
+	else
+		echo "Run: apt-get install rsync file"
+	fi
+	exit 3
+fi
+# Put this to 0 if you want use bash -x to debug it
+debug=1
+hestia_package=default
+tmp_dir='tmp_dir_da_backup'
+time=$(echo "$time_n_date" | cut -f 1 -d \ )
+date=$(echo "$time_n_date" | cut -f 2 -d \ )
+
+if [ -f "$1" ]; then
+	backup_file="$1"
+fi
+
+if [ -z "$BACKUP_TEMP" ]; then
+	BACKUP_TEMP=$BACKUP
+else
+	echo "File does not exists"
+	exit 1
+fi
+
+delete_tmp() {
+	echo "Removing tmp files"
+	rm -rf /root/${tmp_dir}
+}
+
+tput setaf 3
+echo "#######################################"
+echo "# START WITH IMPORT "
+echo "#######################################"
+tput sgr0
+
+tput setaf 2
+echo "Checking provided file..."
+tput sgr0
+if file $backup_file | grep -q -c "gzip compressed data,"; then
+	tput setaf 2
+	echo "OK - Gziped File"
+	tput sgr0
+	if [ ! -d /root/${tmp_dir} ]; then
+		echo "Creating tmp.."
+		mkdir /root/${tmp_dir}
```

Comment:
We should create the temp folder in /backup/ instead /root/

Reply:
Thank you for the response, i changed the temp folder from `/root/` to `/backup/`